cvmfs/cvmfs_server

```diff
@@ -1523,24 +1523,11 @@ create_whitelist() {
   echo `date -u "+%Y%m%d%H%M%S"` > ${whitelist}.unsigned
   echo "E`date -u --date='+30 days' "+%Y%m%d%H%M%S"`" >> ${whitelist}.unsigned
   echo "N$name" >> ${whitelist}.unsigned
```

Comment:
There is a very similar implementation of this functionality [in the test functions](https://github.com/cvmfs/cvmfs/blob/devel/test/test_functions#L1206). Perhaps it's a good idea to change this piece of code as well to avoid confusion down the road.